src/TMedia.cpp

```diff
@@ -729,13 +762,13 @@ void TMedia::downloadFile(TMediaData& mediaData)
         return;
     }
 
-    if (!mediaData.getMediaFileName().isEmpty() && mediaData.getMediaFileName().contains('/')) {
-        const QString mediaSubPath = qsl("%1/%2").arg(mudlet::getMudletPath(enums::profileMediaPath, mpHost->getName()), mediaData.getMediaFileName().section('/', 0, -2));
+    if (!mediaData.mediaFileName().isEmpty() && mediaData.mediaFileName().contains('/')) {
```

Comment:
Here (and elsewhere) even single characters need to be marked up as translatable/not-translatable when they are applied to `QString`s...
```suggestion
    if (!mediaData.mediaFileName().isEmpty() && mediaData.mediaFileName().contains(QLatin1Char('/'))) {
```

Reply:
Done